int32 	formation_index
int32 	cellID